io.github.openWMail.json

```diff
@@ -0,0 +1,62 @@
+ {
+    "app-id": "io.github.openWMail",
+    "base": "io.atom.electron.BaseApp",
+    "base-version": "stable",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "1.6",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "/app/openwmail/openWMail",
+    "separate-locales": false,
+    "rename-desktop-file": "openwmail.desktop",
+    "rename-icon": "openWMail",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=wayland",
+        "--socket=x11",
+        "--socket=pulseaudio",
+        "--share=network",
+        "--device=dri",
+        "--filesystem=xdg-download",
+        "--talk-name=org.freedesktop.Notifications"
+    ],
+    "modules": [
+        {
+            "name": "openWMail",
```

Comment:
It would be good to try to build it from source using helpers from flatpak-builder-tools. It's not mandatory but at some point will be for new apps.

Reply:
For sure.  Getting all npm/yarn dependencies installed is the hardest part with electron apps.  Any known electron apps that build from source? I'll take a look.  I would like to do that to this one and the Rocket.Chat flatpak both.

Reply:
I think io.github.betaflight.BetaflightConfigurator is electron app that builds from source.